yugabyte/run-jepsen.py

```diff
@@ -54,32 +54,41 @@
     {
         "start_version": "1.3.1.0",
         "tests": [
-            "ycql/counter",
-            "ycql/set",
-            "ycql/set-index",
-            "ycql/bank",
-            "ycql/bank-inserts",
-            "ycql/long-fork",
-            "ycql/single-key-acid",
-            "ycql/multi-key-acid",
-
-            "ysql/counter",
-            "ysql/set",
-            "ysql/bank",
-            "ysql/bank-contention",
-            "ysql/bank-multitable",
-            "ysql/long-fork",
-            "ysql/single-key-acid",
-            "ysql/multi-key-acid",
-            "ysql/append",
-            "ysql/append-si",
-            "ysql/default-value",
+            # YCQL snapshot isolation
+            "ycql/si.counter",
+            "ycql/si.set",
+            "ycql/si.set-index",
+            "ycql/si.bank",
+            "ycql/si.bank-inserts",
+            "ycql/si.long-fork",
+            "ycql/si.single-key-acid",
+            "ycql/si.multi-key-acid",
```

Comment:
As far as I'm aware, `YCQL` doesn't have a concept of configurable isolation, why do we need it here then?